core/modules/layout/layout.admin.inc

```diff
@@ -1909,6 +1910,54 @@ function layout_enable_layout($status, Layout $layout) {
   backdrop_goto('admin/structure/layouts');
 }
 
+/**
+ * Menu callback; Enable or disable a layout template.
+ */
+function layout_enable_layout_template($status, $template_name) {
```

Comment:
I liked better how @jenlampton has named the respective function that enables/disables file types, and how the parameter for the thing being toggled comes before the status being toggled to. Can we please change this to:

```suggestion
function layout_layout_template_toggle_enable($template_name, $status) {
```

Reply:
We dont have a convention for this, I'm OK with `layout_enable_layout_template()`.

Reply:
> Can we please change this to:

Yeah I wasn't crazy about the parameter order either. I made it that order because it matches the function directly above it, which is:

```
function layout_enable_layout($status, Layout $layout) {
```

So that's why I did the same thing for a layout template:
```
function layout_enable_layout_template($status, $template_nname) {
```

I think we could flip both though. It's a menu callback that isn't likely used anywhere else.